dolphinscheduler-scheduler-plugin/dolphinscheduler-scheduler-quartz/src/main/java/org/apache/dolphinscheduler/scheduler/quartz/QuartzScheduler.java

```diff
@@ -49,7 +48,7 @@ public class QuartzScheduler implements SchedulerApi {
     @Autowired
     private Scheduler scheduler;
 
-    private final ReadWriteLock lock = new ReentrantReadWriteLock();
+    private final ReentrantLock lock = new ReentrantLock();
```

Comment:
We don't need this lock.

Reply:
@ruanwenjun There has any other needs to be improved? Thx!